modules/preparing-an-initial-cluster-deployment-for-mce-disconnected.adoc

```diff
@@ -70,11 +70,11 @@ $ oc mirror --dest-skip-tls --config ocp-mce-imageset.yaml docker://<your-local-
 
 . Update the registry and certificate in the `install-config.yaml` file:
 +
-.Example `imageContentSources.yaml`
+.Example `imageDigestSources.yaml`
```

Comment:
Is this related to oc-mirror, I am not sure if in 4.14 oc-mirror migrated to use imageDigestSources? Could you double check with oc-mirror team?